client/src/app/site/pages/meetings/modules/poll/services/poll.service/poll.service.ts

```diff
@@ -111,18 +111,21 @@ export abstract class PollService {
         return tableData;
     }
 
-    public getChartLabels(poll: PollData): string[] {
+    public getChartLabels(poll: PollData, excludeYNALabels: boolean = false): string[] {
         const fields = this.getPollDataFields(poll);
         return poll.options.map(option => {
             const votingResults = fields.map(field => {
                 const voteValue = option[field] as number;
                 const votingKey = this.translate.instant(this.pollKeyVerbose.transform(field));
                 const resultValue = this.pollParseNumber.transform(voteValue);
                 const resultInPercent = this.getVoteValueInPercent(voteValue, { poll, row: option });
-                let resultLabel = `${votingKey}: ${resultValue}`;
+                let resultLabel = `${resultValue}`;
+                if (!excludeYNALabels) {
+                    resultLabel = `${votingKey}: ${resultLabel}`;
+                }
 
                 // 0 is a valid number in this case
-                if (resultInPercent !== null) {
+                if (resultInPercent !== null && resultInPercent !== ``) {
```

Comment:
Is `undefined` allowed? If not, you can do the following:
```suggestion
                if (!resultInPercent) {
```